stockflux-securities-master/src/App.js

```diff
@@ -8,11 +8,22 @@ import { Redirect, Route, Switch } from "react-router-dom";
 import "./App.css";
 
 const App = () => {
+
+  //hard coded prop to be replaced with securities object to be edited
+  const props = {
+    securityId: "FOO:BAR"
+  };
+
   return (
     <div>
       <Components.Titlebar />
       <Switch>
         <Route exact path="/" component={SecuritiesTable} />
+        <Route
```

Comment:
I think we need to add in some kind of path parameter placeholder here to be able to accept a `securityId` that we can provider when we create links to this route.

Reply:
Okay I've managed to get this working now